tests/vhost_user_blk_test_server.c

```diff
@@ -110,8 +110,10 @@ struct request {
 /*
  * Zero page used for write-zeroes requests
  */
-_Alignas(PAGE_SIZE)
-static uint8_t zero_page[PAGE_SIZE];
+#define ZERO_PAGE_SIZE (64 * 1024)
```

Comment:
Hmm, why this size specifically? Can we just allocate the zero page dynamically, after we determine the host page size instead? E.g. by using `mmap` directly

Reply:
fixed